apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/RODViewPoolCandidatePage.tsx

```diff
@@ -0,0 +1,523 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import UserCircleIcon from "@heroicons/react/24/outline/UserCircleIcon";
+import HandRaisedIcon from "@heroicons/react/24/outline/HandRaisedIcon";
+import ArrowRightCircleIcon from "@heroicons/react/24/solid/ArrowRightCircleIcon";
+import ExclamationTriangleIcon from "@heroicons/react/24/outline/ExclamationTriangleIcon";
+
+import {
+  NotFound,
+  Pending,
+  ToggleGroup,
+  Accordion,
+  Heading,
+  Sidebar,
+  CardBasic,
+  Button,
+  Link,
+} from "@gc-digital-talent/ui";
+import { commonMessages, getLocalizedName } from "@gc-digital-talent/i18n";
+import { notEmpty } from "@gc-digital-talent/helpers";
+
+import PageHeader from "~/components/PageHeader";
+import {
+  User,
+  Scalars,
+  useGetPoolCandidateSnapshotQuery,
+  PoolCandidate,
+  Maybe,
+  Pool,
+} from "~/api/generated";
```

Comment:
This has been deprecated so new features should only be using the `@gc-digital-talent/graphql` package. You can follow [the documentation](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/documentation/graphql.md) and ping me if you have any question.

Reply:
Fixed by [39b7ce5](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8853/commits/39b7ce5e914120848d26ab04a8e94084bed00af3)